cubical/data_handler/ms_tile.py

```diff
@@ -771,7 +771,7 @@ def load(self, load_model=True):
 
             if self.dh._apply_flags:
                 flagcol = self.dh.fetchslice("FLAG", subset=table_subset)
-                flagcol[:] = np.logical_or.reduce(axis=-2)[:,:,np.newaxis]
+                flagcol[:] = np.logical_or.reduce(flagcol, axis=-1)[:,:,np.newaxis]
```

Comment:
Haha nice catch @JSKenyon. This means that people who don't use bitflags (cough @bennahugo cough) had flags propagate across an entire frequency channel of a tile (rather than across 4 corrs as was intended). Oh well, RFI is largely frequency-based :)

Reply:
Made my local test barf completely so I am happy that my intuition was correct.